recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.6.26" %}
+{% set version = "1.6.28" %}
```

Comment:
@lexual I guess we should update the pinning to `libpng >=1.6.28,<1.7`in https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L49

That way this version will be used to build other packages.